src/ensembl/src/shared/components/question-button/QuestionButton.tsx

```diff
@@ -7,6 +8,7 @@ import styles from './QuestionButton.scss';
 
 type Props = {
   helpText: React.ReactNode;
+  style: string;
```

Comment:
Several points:

1) Do we want flexibility for this button (i.e. is there a reasonably high probability that it will have to have different sizes in different places), or can we be reasonably sure that there will be only a handful of modifications? If the latter is correct, then this approach is fine; but if we want to ensure flexible styling, then passing additional classes to modify the appearance of the question button (see [ImageButton](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/image-button/ImageButton.tsx#L24) as an example) will be a better approach.

2) If we are reasonably certain that there will be only a handful of question button variants, then we should explicitly type them here, for the benefit of future reader, so that `style` is not any old string, but only a member of an enum:

```
enum QuestionButtonVariant {
  SMALL = 'small'
}

type Props = {
  ...blah,
  style: QuestionButtonVariant
}
```

3) The `style` attribute has a special meaning in react — it can be used to pass styles to native html elements using an object of CSS properties. So it's better to name this property (if this solution is eventually chosen instead of passing modifying classes) `type`, or `kind`, or `variant`, or something of this sort.

Reply:
Also, notice that this pull request created type errors:

![image](https://user-images.githubusercontent.com/6834224/68410537-c887b500-0180-11ea-8edb-028367a91743.png)

Reply:
Good points. I did originally do it with an enum, but thought it might be overkill. I'm happy to rename the variable - I was stumped on what else to call it!

Reply:
Changes made as discussed, and it's compiling cleanly - could you have another look?